src/wsproxy.ml

```diff
@@ -45,7 +46,15 @@ let start path handler =
          Lwt_unix.close fd_sock2 >>= fun _ ->
```

Comment:
<for line 26>: Is this running as a daemon in the background, with systemd? Do we restart it in case of failures? Because in that case we should consider doing the same thing as in https://github.com/xapi-project/nbd/pull/55 to avoid having to wait for a minute before restarting it.